File: codewars/Duplicate Encoder/duplicateEncode.js
function duplicateEncode(word) {
    word = word.toLowerCase();
    let result = '';
    for (let i = 0; i < word.length; i++) {
      if (word.lastIndexOf(word[i]) === word.indexOf(word[i])) {
        result += '(';
      } else {
        result += ')';
      }
    }
    return result;
  }

File: codewars/Duplicate Encoder/duplicate_encode.py
def duplicate_encode(word):
    word = word.lower()
    count = {}

    for char in word:
        count[char] = count.get(char, 0) + 1

    result = ""
    for char in word:
        result += "(" if count[char] == 1 else ")"

    return result